### ISSUE TITLE
[Adaptive Cards - Samples - Activity update] - Narrator is incorrectly announcing as "selected" for all the left navigation control present on the 'Samples' page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9307]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Screen Reader: Narrator, NVDA

**Repro Steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the above URL on the Edge Dev browser.
3. Navigate to the 'Samples' control present on the header section of the page using tab key and activate using enter key.
4. Navigate to the left navigation control using tab key.
5. Verify that, narrator is announcing state information as "selected" for all the left navigation control present on the 'Samples' page or not.

**Actual Result:**
Narrator is incorrectly announcing state information as "selected" for all the left navigation control present on the 'Samples' page.

**Note:**
Same issue does not replicate with NVDA.

**Reproducibility:**
Same issue replicates for all the left navigation control present on the 'Schema Explorer' page.

**Expected Result:**
Narrator should announce the "selected" state information only for the currently selected item in the left navigation on the Samples page. Other navigation controls should be announced with their correct default state. As users move through the navigation, Narrator should accurately update and announce focus and selection changes. This ensures clear and predictable navigation for screen reader users.

**User Experience:**
Visually impaired users who rely on screen readers to navigate the web content will be impacted if all left navigation controls are announced as "selected." Users may be unable to determine which navigation item is actually active. This can cause confusion and increase cognitive load while navigating the page. As a result, users may have difficulty understanding the current context or reaching the intended section efficiently.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/78d13877-82cf-42b5-86f3-7949d7e0fe42)

https://github.com/user-attachments/assets/808413b1-249b-4447-8a24-423d745c010a

### Card JSON

```JSON
NA
```
